src/Engine/PartyEnums.h

```diff
@@ -174,7 +174,8 @@ enum class PartyAction : uint32_t {
 };
 using enum PartyAction;
 
-enum class PartyAlignment: int32_t {
+enum class PartyAlignment : int32_t {
+    PartyAlignment_None = -1,
```

Comment:
Let's do `std::optional` in `SetUserInterface` instead? Having an additional value here implies it can be a valid enum value, but it's really not, it's only used in `SetUserInterface`.